setup/setup.go

```diff
@@ -77,6 +77,11 @@ const (
 	APIRecoverRBT                       string = "/api/recover-token"
 	APIValidateTokenChain               string = "/api/validate-token-chain"
 	APIValidateToken                    string = "/api/validate-token"
+	APIDeployNFT                        string = "/api/deploy-nft"
+	APITransferNFT                      string = "/api/transfer-nft"
+	APIGetNFTTokenChain                 string = "/api/get-nft-token-chain"
```

Comment:
A better endpoint name could be `/api/dump-nft-token-chain`. Refer this [comment](https://github.com/rubixchain/rubixgoplatform/pull/229/files#r1802492977)